manifests/dex.cfg

```diff
@@ -21,10 +21,10 @@ connectors:
   id: ldap
   name: LDAP
   config:
-    host: {{.ldap.host}}:636
+    host: {{.ldap.host}}:{{.ldap.port}}
     insecureSkipVerify: true
-    bindDN: $AD_USERNAME
-    bindPW: $AD_PASSWORD
+    bindDN: {{.ldap.bindDN}}
```

Comment:
We don't want to inject the username and password directly into the configmap as they are sensitive, we, therefore, store them in secrets, load them into DEX as environment variables where DEX does the replacement.